tests/PerformanceTest.php

```
@@ -0,0 +1,116 @@
+        PerformanceUtils::startPageCache($page);
+        echo "Test Page Static Body Content";
+        PerformanceUtils::endPageCache($page);
+
+        // Assert file is successfully written
+        $this->assertFileExists($cacheFile);
+        $this->assertStringEqualsFile($cacheFile, "Test Page Static Body Content");
+
+        // Clear output buffering and run page cache start to verify it exits (Simulate cache HIT)
+        // Since startPageCache exits on HIT, we can inspect filemtime to ensure it matches
+        $this->assertGreaterThanOrEqual(time() - 5, filemtime($cacheFile));
+    }
```

PHPMD (2.15.0)

[error] testStaticPageCaching accesses the super-global variable $_SERVER. (undefined)

(Superglobals)